engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/NkGlobalParameters.scala

```diff
@@ -24,6 +26,17 @@ case class NkGlobalParameters(buildInfo: String,
       "user" -> processVersion.user
     ).filterNot(_._2 == null).asJava
   }
+
+  // This function is used in (not-legacy) `MetricUtils` to add additional tags to all types of metrics.
+  // If you add new tag, remember that it will also influence the measurement name. In order to avoid it,
+  // make sure you modify `telegraf.conf` accordingly.
+  def toTags: Map[String, String] = {
```

Comment:
let's move this logic to MetricUtils, here we shouldn't put logic connected to metrics, just the data?

Reply:
Good point, moved